src/patterns/utilities/single-property/color.hbs

```diff
@@ -16,46 +16,47 @@ description: These utilities control CSS properties related to color and backgro
     </tr>
   </thead>
   <tbody>
-    <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-Color--red</td>
-      <td>Sets the font color to red.</td>
-    </tr>
     <tr>
       <td class="sprk-u-FontWeight--bold">.sprk-u-Color--black</td>
       <td>Sets the font color to black.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-Color--white</td>
-      <td>Sets the font color to white.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-Color--blue</td>
+      <td>Sets the font color to blue.</td>
     </tr>
     <tr>
       <td class="sprk-u-FontWeight--bold">.sprk-u-Color--gray</td>
       <td>Sets the font color to gray.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-Color--red-mid</td>
-      <td>Sets the font color to red-mid.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-Color--green</td>
```

Comment:
What were these changes in here for? Looks like moving the color utilities around?

Reply:
I reordered the colors to match the order in the design doc.